src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -313,8 +314,10 @@ trait EntityComponent {
         recursiveGetEntityReferenceIds(idSet, idSet)
       } flatMap { ids =>
         DBIO.sequence(ids.map { id =>
-          val sql = sql"""#$baseEntityAndAttributeSql where e."id" = ${id}""".as[EntityListResult]
```

Comment:
this should work except for the double quotes. The fancy string stuff is not required. How about
val sql = sql"""#$baseEntityAndAttributeSql where e.#${quoteIdentifier("id")} = ${id}""".as[EntityListResult]